packages/@ember/-internals/runtime/lib/mixins/array.js

```diff
@@ -853,6 +853,15 @@ const ArrayMixin = Mixin.create(Enumerable, {
     object that will be set as "this" on the context. This is a good way
     to give your iterator function access to the current object.
 
+    Example Usage:
+
+    ```javascript
+    const food = Ember.A().addObjects([{food: 'apple', isFruit: true}, {food: 'bread', isFruit: false}, food: 'banana', isFruit: true}]);
```

Comment:
Is this line too long?

Reply:
I would format it so that there is one object per line.
More importantly, if you want to create an EmberArray with items, you can do `Ember.A([1,2,3])`, you don't need to create the array and add objects. The other methods assume that prototype extensions are enabled, so in this example you can define a regular array and  it will work in applications:

```suggestion
    const food = [
      { food: 'apple', isFruit: true},
      { food: 'bread', isFruit: false },
      { food: 'banana', isFruit: true }
    ];
```

Reply:
Awesome, thank you!